terraform/groups/ecs-service/main.tf

```
@@ -115,4 +114,37 @@ resource "aws_security_group" "ecs_sg" {
   name        = "${local.service_name}-${var.environment}-ecs-sg"
   description = "Security group for ECS service"
   vpc_id      = data.aws_vpc.vpc.id
-}
\ No newline at end of file
+}
+
+resource "aws_lb_target_group" "ollama_target_group" {
+  name     = "ollama-target-group"
```

I'm not sure if the target group should be parameterised by environment.
It seems like [other are](https://github.com/companieshouse/bankrupt-officer-search-stack/blob/a70d69e9f73852e350299997f6b7bb9f9761fd8a/groups/stack/module-ecs-services/bados-web.tf#L62), but I don't know if this is relevant to this situation. 

```suggestion
  name     = "${local.service_name}-${var.environment}-target-group"
```